### PR TITLE
Remove react-datepicker package that's unused for the past 2 years, lol

### PR DESCRIPTION
**Pull request by kangax**

### CODE REVIEW

File: src/external/index.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-// import reactselect from './reactselect.scss';
-import reactdatepicker from './reactdatepicker.scss';
-
-const style = {
-  // reactselect,
-  reactdatepicker,
-};
```
Comment (kangax) [inline src/external/index.js]: need to check that we're not using any of this in SS

Comment (kangax) [reply src/external/index.js]: Checked, unused

Comment (maniator) [reply src/external/index.js]: All of the datepickers look kosher in ss?

File: docs/gitbook/setup/README.md [hunk 2]
```diff
@@ -5,8 +5,7 @@ To get up and running developing with Plasma, follow these instructions:
 #### Important note
 
 ````
-Select and Datepicker components do not work right now.
-Use react-select and datepicker directly in Spacestation.
+Select component does not work right now. Use react-select directly in Spacestation.
```
Comment (maniator) [inline docs/gitbook/setup/README.md]: Can we remove the select code as it is not used anywhere (and all commented out where it is used)

**Pull request merged**